dev/Installer/src/install.h

```diff
@@ -0,0 +1,13 @@
+#pragma once
+#include "pch.h"
+#include "packages.h"
+
+struct PackageProperties
+{
+    wil::unique_cotaskmem_string full_name;
+    wil::unique_cotaskmem_string family_name;
+    winrt::Windows::System::ProcessorArchitecture architecture = winrt::Windows::System::ProcessorArchitecture::Unknown;
+    UINT64 version = 0u;
```

Comment:
[External]
Nit: Can we use `std::uint64_t` here?

Reply:
Fixed